### ISSUE TITLE
Installation won't complete > fCGI conflict?

### ISSUE THREAD

**Issue by siniztr** [magento/magento2#17026]
### Preconditions
Doing new install on an existing Linux Ubuntu 17.10 with apache2.
1. Uploaded files via sFTP to web folder
2. Proceeded to Web install and made user and database via PHPmyAdmin

### Steps to reproduce
1. After uploading web package (with samples) downloaded from magento.com with sFTP to web folder, started up Web install script 
2. Did install with all modules enabled. 
3. After entering all required information for the install, the installation process begins and the bar starts going

### Expected result
1. I was hoping that the install would complete successfully.

### Actual result
1. Install hangs at 74%, or after the installation of the final module. 
Error from Apache2 logfiles:

> [proxy_fcgi:error] [pid 1547:tid 140267736053504] [client 84.245.15.8:51073] AH01071: Got error 'PHP message: PHP Fatal error: Uncaught Exception: Warning: ini_set(): Cannot find save handler 'db' in /var/www/youtube/magento/vendor/magento/framework/Session/SessionManager.php on line 563 in /var/www/youtube/magento/vendor/magento/framework/App/ErrorHandler.php:61\nStack trace:\n#0 [internal function]: Magento\Framework\App\ErrorHandler->handler(2, 'ini_set(): Cann...', '/var/www/youtub...', 563, Array)\n#1 /var/www/youtube/magento/vendor/magento/framework/Session/SessionManager.php(563): ini_set('session.save_ha...', 'db')\n#2 /var/www/youtube/magento/vendor/magento/framework/Session/SessionManager.php(184): Magento\Framework\Session\SessionManager->initIniOptions()\n#3 /var/www/youtube/magento/generated/code/Magento/Backend/Model/Auth/Session/Interceptor.php(167): Magento\Framework\Session\SessionManager->start()\n#4 /var/www/youtube/magento/vendor/magento/framework/Session/SessionManager.php(130): Magento\Backend\Model\Auth\Session\Interceptor->start()\n#5 /var/www/youtube/magento/vendor/magento/module-b...\n', referer: depreciated for privacy reasons

Current setup:
Ubuntu 17.10 with Apache2, mySQL, http/2, LetsEncrypt and fCGI. 
PHP modules enabled:
apc, apcu, bcmath, calendar, Core, ctype, curl, date, dom, exif, fileinfo, filter, ftp, gd, gettext, hash, iconv, imagick, imap, intl, json, ldap, libxml, mbstring, mcrypt, memcache, mysqli, mysqlnd, openssl, pcntl, pcre, PDO, pdo_mysql, pdo_sqlite, Phar, posix, pspell, readline, recode, Reflection, session, shmop, SimpleXML, soap, sockets, SPL, sqlite3, standard, sysvmsg, sysvsem, sysvshm, tidy, tokenizer, wddx, xml, xmlreader, xmlrpc, xmlwriter ,xsl ,Zend OPcache ,zip ,zlib

**Comment by magento-engcom-team** [comment]
Hi @siniztr. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me {$VERSION} instance` 

where `{$VERSION}` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (2.2-develop +). 
For more details, please, review the [Magento Contributor Assistant](https://github.com/magento/magento2/wiki/Magento-Contributor-Assistant) documentation.

@siniztr do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by siniztr** [comment]
@magento-engcom-team give me 2.2.5 instance

**Comment by magento-engcom-team** [comment]
Hi @siniztr. Thank you for your request. I'm working on Magento 2.2.5 instance for you

**Comment by magento-engcom-team** [comment]
Hi @siniztr, here is your [Magento instance](https://i-17026-2-2-5.engcom.dev.magento.com).
Admin access: [https://i-17026-2-2-5.engcom.dev.magento.com/admin](https://i-17026-2-2-5.engcom.dev.magento.com/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by siniztr** [comment]
As the issue is related to installation, I am not able to reproduce in the Magento instance provided.

**Comment by siniztr** [comment]
@magento-engcom-team No can't reproduce

**Comment by ghost** [comment]
Hi @siniztr which version of magento you tested? and php version ?

**Comment by siniztr** [comment]
I installed the latest version offered by both the website and github. PHP version and additional mods are posted in initial post. Tested on PHP7.1 at first, then with 7.2, and with (among others) fCGI, HTTP/2, OPCache, memcache and php-FPM. I am running test linux boxes on VMWare vsphere where I test installations on a "real" production level for clients, before actually purchasing a VPS server and putting the install online.
